client/app/calendar/tasks/components/DateField.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react'
+import { format } from 'date-fns'
+import { Control, Controller, FieldValues, Path } from 'react-hook-form'
+import { BiCalendarAlt } from 'react-icons/bi'
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  Calendar,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/_components/ui'
+
+type DateFieldProps<TFormValues extends FieldValues> = {
+  control: Control<TFormValues>
+  name: Path<TFormValues>
+}
+
+export const DateField = <TFormValues extends FieldValues>({
```

Comment:
export default? Please check other components too!

Reply:
Fixed it!
https://github.com/HAK2024/fairy-share/pull/80/commits/718f254b1f61aa7177556fa86e2119e7b40b106a